readme.md

```diff
@@ -0,0 +1,20 @@
+#BSO-Project
+
+**Building Semantic Ontology Project** is an open-source initiative aimed to ease the process of tagging realtime data in buildings. It is based on the open-source [Project Haystack](https://project-haystack.org/) which defines roughly 200 standard names and relations for describing building devices and points.
+
+You can access directly the interface [bso-project.org](https://bso-project.org/#/ontology-creator)
+
+##Goal
+The goal of the project is to describe *what is* the data. We might call a point sometimes **_temperature_** or **_exhaust temperature_** or **_exhaust_temperature_**. Well, with BSO-Project, il will be **Ventilation** -> **AHU** -> **Duct Sensor** + **Exhausting** + **Measure** + **Temperature** + **Air**.
+
+##Features
+BSO-Project provides 3 main features on the top of it:
+
+* A type based naming that we call Nodes. They represent abstract element (temperature, air etc.) or real devices. They can contain several tags.
```

Comment:
A type based naming that we call Nodes. => I would have call it "elements", we don't understand at first sight